### ISSUE TITLE
Race, Feat, and Spell Formatting

### ISSUE THREAD

**Issue by Tarkisflux** [SqueeG/awesomeTome#33]
I've been busy with single column formatting, and am happy enough with it to ask for feedback. Here's a description, but you can see all of it in the current pdf anyway:

Races have fluff at the top, a few tables of standard data in multicolumn (they're small), and special abilities at the end. I think it's a good use of space and should display well in single column.

Feats are just regular single column. There's a few spacing issues where feats are overseparated when text gets right to the end of the line in the previous feat, but these should otherwise be set.

Spells have standard data floating on the right, with spell text on the left and underneath if it gets long enough. Given the preference for single column, I'm not sure there's a better way to set that up that doesn't waste lots of space. We can get 4-5 spells per page in this format, though short spells have a fair amount of wasted whitespace on the text side. We can fit more if we're willing to cut standard data out of the box (like not showing action unless it's not standard action, or not showing SR unless it's allowed, or cutting duration, etc.).

Feedback and thoughts appreciated.

**Comment by Tarkisflux** [comment]
Nevermind on feat spacing. There's a difference between a leading \newline and a trailing \ I guess. Feat spacing is consistent now, and we can make minor tweaks to the gaps between items if we want.

**Comment by Bihlbo** [comment]
I really like the aasimar template, and applied that to dwarf, elf, and
gnome. If I didn't sync my fork properly, it's because git is bloody
incomprehensible. I'm not sure the best way to handle the formatting of the
gnome's stupid sla block - mimicing the aasimar's single sla didn't quite
get me there. None of the original races have descriptions in the SRD. How
are we gonna tackle that? Tell people to look in the PH? Find non-WotC
descriptions that we don't have to worry about using?

On single column formatting:
While I find two columns much more readable when in print, I suppose if we
have to accommodate small screen reading and it's a lot easier to do, then
I'll be happy with single column.

Except for feats and spells. Those really aught to be on a two-column page,
or their display format should be altered away from what everyone is
already used to, in order to accommodate the width. I don't think the
concerns about 2-col apply to feats and spells, so I don't see a reason to
do it 1-col other than expediency. And as far as that goes, the plan "Put
them in formatted, we can apply columns later" sounds perfectly fine, as
long as that is possible within latex.

-Bill

**Comment by Tarkisflux** [comment]
@ExplosiveRunes is responsible for the tables and the basic race format. The original plan was to write a bunch of fluff for each race to fill in a column so that the tables could float on the right, but I thought it was starting to look too much like a two-column layout and so I shuffled things around a bit. I'm sure he'll let me know if it looks like shit and I killed his baby and I'm a terrible person for doing so.

As for the non-SRD descriptions, we'll need to write them like we need to write a few other non-SRD sections. Or we could just strip them out for every race and not worry about them at all - that's an option too I just think it's a worse one. Default fluff and whatnot.

I'd take a look at the gnome problems, but it looks like you committed to your fork rather than the master branch (SqueeG's). There's a way to push your stuff back into the main branch (possibly called a pull request), but I'm not sure how to trigger it. @SqueeG should be able to assist though, soon as he wanders in here.

**Comment by Tarkisflux** [comment]
Yeah, you want to put in a pull request. There's a button on the right side of the website with a couple of arrows making an almost square, right under the issues button, that you can use to start that. I'd play with it and try to make it work, because even if you break something we can just fix it, but that's me.

Anyway, after looking at your branch, it looks like you're a version behind the race formatting I worked up. I can grab your race files and import them into the revised stuff, and then you can sync your branch with the master to see the changed format. And hopefully you like the newer version just as much or more. And if not, we can always go back.

**Comment by SqueeG** [comment]
@Bihlbo Which Git client are you using? github for windows makes things really easy for git beginners.

With Git you just need to remember that there are three steps to `commit`ing.

(4 steps including zero)

**[rule 0]** _Sync and make sure you have the latest changes_
1. Pick the changed files you want to commit and wrote a brief description about what changed
2. `commit` those changes
3. `push`/ `sync` those changes

**Comment by Bihlbo** [comment]
Finally finished a big project at work and have more time. Made a description for the dwarves and updated that file to fit the new race template. Feel free to pick apart that description - I made up most of it based on Tome, the 3.5 PH, and my own interpretation of what those two views do to the race.

SqueeG, I'm using 1.3.3.1 Github for Winders. Thanks to all y'all I'm getting the hang of it.
